packages/perseus-editor/src/__stories__/editor.stories.tsx

```diff
@@ -11,6 +11,8 @@ import {registerAllWidgetsAndEditorsForTesting} from "../util/register-all-widge
 
 import type {PerseusRenderer} from "@khanacademy/perseus";
 
+import "../styles/perseus-editor.less";
```

Comment:
This was a big part of the fix. I'm not 100% sure why it worked before, but we have to ensure we're actually importing the `.less` file that provides the style. I suspect it's a bundling issue where the new Webpack bundler (v5 compared to v4) skips the entry `index.ts` file which has the `.less` import. 🤷‍♂️